pkg/sysfs/system.go

```diff
@@ -625,15 +581,6 @@ func (sys *system) discoverCPU(path string) error {
 
 	sys.cpus[cpu.id] = cpu
 
-	if (sys.flags & DiscoverCache) != 0 {
-		entries, _ := filepath.Glob(filepath.Join(path, "cache/index[0-9]*"))
-		for _, entry := range entries {
-			if err := sys.discoverCache(entry); err != nil {
```

Comment:
nitpick: If we're eliminating unused code (this way) then we should also remove `discoverCache()`.

Reply:
Especially since it is broken (recently fixed in nri-plugins).

Reply:
Ach, thanks for pointing this out. Changed now (i.e. `discoverCache` dropped)